aequilibrae/paths/linear_approximation.py

```diff
@@ -197,22 +197,14 @@ def calculate_biconjugate_direction(self):
         z_ = {}
 
         for c in self.traffic_classes:
-            x_[c.__id__] = np.sum(
-                (
-                    self.step_direction[c.__id__].link_loads[:, :] * self.stepsize
-                    + self.previous_step_direction[c.__id__].link_loads[:, :] * (1.0 - self.stepsize)
-                    - c.results.link_loads[:, :]
-                ),
-                axis=1,
-            )
-
-            y_[c.__id__] = np.sum(c._aon_results.link_loads[:, :] - c.results.link_loads[:, :], axis=1)
-            z_[c.__id__] = np.sum(self.step_direction[c.__id__].link_loads[:, :] - c.results.link_loads[:, :], axis=1)
+            sd = self.step_direction[c.__id__].link_loads[:, :]
```

Comment:
@janzill , This part has always been a pain in the neck with respect to formatting, so I just broke it into getting the components and then doing the math.